spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -847,13 +845,12 @@ public interface AnnotatedTypeConfigurator<T> {
 
 * `getAnnotated()` returns the original `AnnotatedType` with which this configurator was initialized
 * `add()` adds an annotation to the configured `AnnotatedType`
-* `remove()` methods remove all annotation from the configured `AnnotatedType`
-* `removeAll()` remove all existing annotation on the configured `AnnotatedType`
```

Comment:
Just noticed this - since we have kept the `removeAll` method, this line should stay, right?